pkg/proxy/iptables/proxier.go

```diff
@@ -300,15 +315,26 @@ func (scm *serviceChangeMap) update(namespacedName *types.NamespacedName, previo
 	return len(scm.items) > 0
 }
 
+// ipPort returns a string of the form "<ip>:<port>" (for IPv4)
+// or "[<ip>]:<port>" (for IPv6) for a given IP address and port
+func ipPort(ipStr string, port int) string {
+	ip := net.ParseIP(ipStr)
```

Comment:
Do we know that ipStr will always be an IP address (and not a name)? Is it guaranteed to always be valid?

Reply:
@thockin: Yes, that will do the trick, thanks!